automated/linux/ltp/skipfile-lkft.yaml

```diff
@@ -18,13 +18,31 @@
 # tests:
 #   List of tests to skip
 
+globals:
+  - boards: &all_boards
+    - bcm2711-rpi-4-b
+    - dragonboard-410c
+    - dragonboard-845c
+    - e850-96
+    - hi6220-hikey-r2
+    - juno-r2
+    - qemu
+    - qemu-arm
```

Comment:
qemu and qemu-arm isn't used as device-types.
in the lab qemu_arm, qemu_arm64, qemu_x86_64 and qemu_i386

Reply:
Corrected this :+1: